spot-electron/package.json

```diff
@@ -1,6 +1,7 @@
 {
     "author": "Jitsi",
     "build": {
+        "productName": "Jitsi Spot",
```

Comment:
Isn't it "Spot" that's how the mobile app is currently called.

Reply:
Ok